core/src/test/resources/logback-test.xml

```diff
@@ -1,20 +1,3 @@
-<!--
```

Comment:
We normally keep license headers on XML files, see https://github.com/HotelsDotCom/waggle-dance/blob/master/waggle-dance-integration-tests/src/test/resources/log4j2.xml

Reply:
This was added manually I assume. This project uses a different plugin for the license and it was only configured for java files. I spent a bit of time configuring it for xml files but I would've needed to provide another one with xml comment syntax around it (the LICENSE.txt contains java comment syntax). I gave up. We can do it in another PR. See #176